A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_FTCL.js

```diff
@@ -186,7 +194,89 @@ let A320_Neo_LowerECAM_FTCL;
             } else {
                 this.sec3Shape.setAttribute("class", "MainShapeWarning");
                 this.sec3Num.setAttribute("class", "Value15Warning");
-            }
+            } 
+
+            if (this.hydraulicsAvailable) {
+                // Show hydraulic numbers and hide everything else
+                this.spoilertext.setAttribute("visibility", "visible");
+                this.spoiler_a5l.setAttribute("visibility", "hidden");
+                this.spoiler_a4l.setAttribute("visibility", "hidden");
+                this.spoiler_a3l.setAttribute("visibility", "hidden");
+                this.spoiler_a2l.setAttribute("visibility", "hidden");
+                this.spoiler_a1l.setAttribute("visibility", "hidden");
+                this.spoiler_a5r.setAttribute("visibility", "hidden");
+                this.spoiler_a4r.setAttribute("visibility", "hidden");
+                this.spoiler_a3r.setAttribute("visibility", "hidden");
+                this.spoiler_a2r.setAttribute("visibility", "hidden");
+                this.spoiler_a1r.setAttribute("visibility", "hidden");
+            } else {
+                // Update spoiler variables
+                const spoilersArmed = SimVar.GetSimVarValue("SPOILERS ARMED", "boolean");
+                const leftSpoilerDeflectPct = SimVar.GetSimVarValue("SPOILERS LEFT POSITION", "percent over 100");
+                const rightSpoilerDeflectPct = SimVar.GetSimVarValue("SPOILERS RIGHT POSITION", "percent over 100");
+                const planeOnGround = SimVar.GetSimVarValue("SIM ON GROUND", "boolean");
+                const eng1_mode = SimVar.GetSimVarValue("GENERAL ENG THROTTLE MANAGED MODE:1", "number");
+                const eng2_mode = SimVar.GetSimVarValue("GENERAL ENG THROTTLE MANAGED MODE:2", "number");
+                const gspeed = SimVar.GetSimVarValue("SURFACE RELATIVE GROUND SPEED", "feet_per_second");
+
+                // Disable hydraulic numbers
+                this.spoilertext.setAttribute("visibility", "hidden");
+
+                // Update left speedbrakes and aileron
+                if (leftSpoilerDeflectPct > 0.07) {
+                    console.log("left aileron pct: " + String(leftAileronDeflectPct));
+                    console.log("left splr pct: " + String(leftSpoilerDeflectPct));
```

Comment:
`console.log` has performance implications, you should remove it in production code

Reply:
Done.